cmd/changes_submit_plan.go

```diff
@@ -336,7 +341,9 @@ func SubmitPlan(cmd *cobra.Command, args []string) error {
 func init() {
 	changesCmd.AddCommand(submitPlanCmd)
 
-	submitPlanCmd.PersistentFlags().String("frontend", "https://app.overmind.tech", "The frontend base URL")
+	addAPIFlags(submitPlanCmd)
+	submitPlanCmd.PersistentFlags().String("frontend", "", "The frontend base URL")
+	_ = submitPlanCmd.PersistentFlags().MarkDeprecated("frontend", "This flag is no longer used and will be removed in a future release. Use the '--app' flag instead.")
```

Comment:
```suggestion
	cobra.CheckErr(submitPlanCmd.PersistentFlags().MarkDeprecated("frontend", "This flag is no longer used and will be removed in a future release. Use the '--app' flag instead."))
```

always panic on setup-errors with cobra/viper.

Reply:
so example usage in https://github.com/spf13/pflag/blob/d5e0c0615acee7028e1e2740a11102313be88de1/flag_test.go#L998 

not great docs, around what the actual error can be, but i put a comment in with what i was able to figure out.

Reply:
> MarkDeprecated only errors if the flag doesn't exist

Yeah, I know. How do I know? misspelling the flag name in the code, leading to silently not having the flag deprecated that I wanted to have deprecated.

The only time this will panic is a programming error, and we'll all be glad to have that, rather than deploying silent corruption.